### PR TITLE
add uniform placeholders and styling to modals

### PR DESCRIPTION
**Pull request by robertjallen**
### Description

**What work was done?**
- add placeholders to all inputs in all modals
- modified some styling for uniformity. 

**Why was this work done?**
- There were some inputs missing placeholders 

### Type of change
- New feature (non-breaking change which adds functionality)

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

File: src/components/forms/AddServiceForm.js [hunk 1]
```diff
@@ -18,6 +26,7 @@ function AddServiceForm({
   serviceTypes,
 }) {
   const [form] = Form.useForm();
+  // console.log("serviceForm", state)
```
Comment (PurdyRad) [inline src/components/forms/AddServiceForm.js:29]: Would say we could remove the console.log here

File: src/components/forms/AddServiceForm.js [hunk 2]
```diff
@@ -93,7 +103,14 @@ function AddServiceForm({
...
           >
-            <Input size="large" min="0" />
+            <Select size="large" placeholder="Select Unit Type">
+              {serviceTypes.map(item => (
+                <Select.Option key={item.id} value={item.id}>
+                  {item.name}
+                </Select.Option>
+              ))}
+            </Select>
+            {/* <Input size="large" min="0" placeholder="Select Service Type"/> */}
```
Comment (PurdyRad) [inline src/components/forms/AddServiceForm.js:113]: Could remove this commented out line if we don't need it anymore

**Review by PurdyRad — COMMENTED** [review]
All in all pretty good, seems the majority of stuff was just some styling here and there. Only real criticism would be the couple lines of commented out code I saw. Other than that good job!

File: src/components/forms/AddServiceForm.js (hunk 1 above)
Comment (robertjallen) [reply src/components/forms/AddServiceForm.js]: got it

**Comment by blackcatwizard** [comment]
it's failing a check? what can we do to fix that?

**Review by Jonathan1600 — APPROVED** [review]
It looks pretty good to me tbh! Other than those extra comments.

**Pull request merged**